theory-test-admission-candidate-app/app/server/controllers/candidateDetailsController.js

```diff
@@ -10,15 +11,18 @@ export default class CandidateDetailsController {
 	checkBooking(req, res) {
 		const admissionsId = uuid();
 		const candidateDLN = req.body.DLN.toString().toUpperCase();
-		this.candidateBookingService.admissionsId = admissionsId;
-		this.candidateBookingService.candidateDLN = candidateDLN;
-		const hasBooking = this.candidateBookingService.checkCandidateBooking();
-		sessionStorage(req, res, 'bookingResult', { hasBooking, admissionsId });
-		if (hasBooking) {
-			sessionStorage(req, res, 'videoTermsAgreed', false);
-			res.redirect('/candidate/prepare-video');
-		} else {
-			res.redirect('/candidate/report-reception');
-		}
+		CandidateCheckBookingService.checkCandidateBooking(admissionsId, candidateDLN, (hasBooking) => {
+			logger.info('GotBooking: ', hasBooking);
+			sessionStorage(req, res, 'bookingResult', { hasBooking, admissionsId });
+			logger.info('Stored in session storage');
+			if (hasBooking) {
+				logger.info('has booking');
+				sessionStorage(req, res, 'videoTermsAgreed', false);
+				res.redirect('/candidate/prepare-video');
```

Comment:
Why is 'candidate/' appearing in these URLs?
The whole app is for the candidate.

Reply:
There's candidate and admin

Reply:
![image](https://user-images.githubusercontent.com/8133611/37515339-651bc9a0-2902-11e8-9c25-97d25576f5bf.png)